tests/unit/amo/middleware/test_responseTime.js

```diff
@@ -130,7 +118,7 @@ describe(__filename, () => {
     });
 
     it('sets up a prefix', () => {
```

Comment:
I know this is old code, but it seems totally reasonable to just combine this test with the one above (for `configures the client`). WDYT?

Reply:
let's do this